src/html/HtmlNodeTag.js

```diff
@@ -42,7 +44,16 @@ const specialTags = {
 const stylesFromClassNames = (classNames = '', styleObj) =>
   classNames.split(' ').map(className => styleObj[className]);
 
-export default ({ auth, attribs, name, cascadingStyle, childrenNodes, onPress }) => {
+type FuncArguments = {
```

Comment:
Looks like we have a mix of `funcArguments` and `FuncArguments` in different places. Could we standardize to one of them? I think the convention is `FuncArguements` but could be mistaken